components/governance/org.wso2.carbon.governance.registry.extensions/src/org/wso2/carbon/governance/registry/extensions/indexers/RXTIndexer.java

```diff
@@ -128,7 +128,7 @@ public void setQName(QName qName) throws GovernanceException {
      * @param elementString     rxt configuration.
      * @throws RegistryException
      */
-    private static void updateTenantsUnboundedFieldMap(String elementString) throws RegistryException {
+    public static void updateTenantsUnboundedFieldMap(String elementString) throws RegistryException {

```

Comment:
It would be better if we can move this method to CommonUtil class and refer it from here.